lib/init/npm-utils.js

```diff
@@ -39,6 +39,34 @@ function findPackageJson(startDir) {
     return null;
 }
 
+/**
+ * Check if the given file exists,
+ * @param   {string} file name or path from cwd
+ * @returns {boolean} indicates the existence of file
+ */
+function isFileExists(file) {
+    return fs.existsSync(path.resolve(process.cwd(), file));
+}
+
+/**
+ *
+ * Returns the name of package manager to use,
+ * @returns {string} - The package manager name
+ */
+function getPackageManager() {
+    const hasLocalYarn = isFileExists("yarn.lock");
+    const hasLocalNpm = isFileExists("package-lock.json") || isFileExists("npm-shrinkwrap.json") || isFileExists("node_modules/.package-lock.json");
+
+    if (hasLocalYarn) {
+        return "yarn";
+    }
+    if (hasLocalNpm) {
+        return "npm";
+    }
+
+    return "npm";
```

Comment:
Why not return `"npm"` directly?

Reply:
looks like a leftover from the refactor in the last commit. There was more logic in between earlier :smile: https://github.com/eslint/eslint/pull/13756#discussion_r504856278 .

Waiting for the issue to be accepted before doing any changes.